test/basic/rds_create/mariadb/rds_mariadb_create_test.go

```diff
@@ -21,19 +20,26 @@ func cleanupAndAssert(t *testing.T, terraformOptions *terraform.Options) {
 	// test / assert all failures here
 }
 
-func TestRDS(t *testing.T) {
+func TestRDSCreateMariaDB(t *testing.T) {
 	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
 		TerraformDir: ".",
 
 		Vars: map[string]interface{}{
-			"organization_id": os.Getenv("ORGANIZATION_ID"),
-			"environment_id":  os.Getenv("ENVIRONMENT_ID"),
-			"aptible_host":    os.Getenv("APTIBLE_HOST"),
-			"database_name":   "testrds-db",
-			"vpc_name":        "testrds-vpc",
+			"organization_id":         os.Getenv("ORGANIZATION_ID"),
+			"environment_id":          os.Getenv("ENVIRONMENT_ID"),
+			"aptible_host":            os.Getenv("APTIBLE_HOST"),
+			"database_name":           "test-create-mariadb-57",
+			"database_engine_version": "5.7",
+			"vpc_name":                "rds-create-vpc-mariadb-57",
 		},
 	})
 	defer cleanupAndAssert(t, terraformOptions)
+
+	err := rds_create.CheckOrRequestVPCLimit()
```

Comment:
do we not want this in an init function thing?

Reply:
yeah probably better to do, good call. i can do that inside of that other package i think